bundle/schema/spec.go

```diff
@@ -0,0 +1,210 @@
+	ControlPlaneService string    `json:"x-databricks-controlplane"`
+	IsAccounts          bool      `json:"x-databricks-is-accounts"`
+	Name                string    `json:"name"`
+}
+
+type Path struct {
+	Node
+	Parameters []Parameter `json:"parameters,omitempty"`
+	Get        *Operation  `json:"get,omitempty"`
+	Head       *Operation  `json:"head,omitempty"`
+	Post       *Operation  `json:"post,omitempty"`
+	Put        *Operation  `json:"put,omitempty"`
+	Patch      *Operation  `json:"patch,omitempty"`
+	Delete     *Operation  `json:"delete,omitempty"`
+}
+
+type fieldPath []string
+
+func (fp fieldPath) String() string {
+	return strings.Join(fp, ".")
+}
+
+// Operation is the equivalent of method
+type Operation struct {
+	Node
+	Wait       *Wait       `json:"x-databricks-wait,omitempty"`
+	Pagination *Pagination `json:"x-databricks-pagination,omitempty"`
+	DataPlane  *DataPlane  `json:"x-databricks-dataplane,omitempty"`
+	Shortcut   bool        `json:"x-databricks-shortcut,omitempty"`
+	Crud       string      `json:"x-databricks-crud,omitempty"`
+	JsonOnly   bool        `json:"x-databricks-cli-json-only,omitempty"`
+
+	// The x-databricks-path-style field indicates whether the operation has a
+	// RESTful path style or a RPC style. When specified, this overrides the
+	// service-level setting. Valid values are "rest" and "rpc". "rest" means
+	// that the operation has a RESTful path style, i.e. the path represents
+	// a resource and the HTTP method represents an action on the resource.
+	// "rpc" means that the operation has a RPC style, i.e. the path represents
+	// an action and the request body represents the resource.
+	PathStyle PathStyle `json:"x-databricks-path-style,omitempty"`
+
+	// The x-databricks-request-type-name field defines the name to use for
+	// the request type in the generated client. This may be specified only
+	// if the operation does NOT have a request body, thus only uses a request
+	// type to encapsulate path and query parameters.
+	RequestTypeName string `json:"x-databricks-request-type-name,omitempty"`
+
+	// For list APIs, the path to the field in the response entity that contains
+	// the resource ID.
+	IdField fieldPath `json:"x-databricks-id,omitempty"`
+
+	// For list APIs, the path to the field in the response entity that contains
+	// the user-friendly name of the resource.
+	NameField fieldPath `json:"x-databricks-name,omitempty"`
+
+	Summary     string           `json:"summary,omitempty"`
+	OperationId string           `json:"operationId"`
+	Tags        []string         `json:"tags"`
+	Parameters  []Parameter      `json:"parameters,omitempty"`
+	Responses   map[string]*Body `json:"responses"`
+	RequestBody *Body            `json:"requestBody,omitempty"`
+}
+
+type Components struct {
+	Node
+	Parameters map[string]*Parameter `json:"parameters,omitempty"`
+	Responses  map[string]*Body      `json:"responses,omitempty"`
+	Schemas    map[string]*Schema    `json:"schemas,omitempty"`
```

Comment:
We already have a `jsonschema` library in the CLI. You can reuse `jsonschema.Schema` here.

Reply:
That certainly makes things even simpler!